3209.json

```diff
@@ -0,0 +1,93 @@
+"SysEndgame_Thanks" : " Vielen dank fürs spielen!\njeden monat kommt neues Zeug.",
+"SysIntermission_HitToStart" : "Mach die Rote Box Kalt indem du sie {0} mal schlägst! damit \nDamit startest du das spiel!",
+"SysDesyncError" : "(Nächstes spiel startet...)",
+"PlaySuccess" : "du hast es gut gemacht!",
+"PlayFailure" : "du hast verkackt.",
+"TutorialHandinessLeft" : "Schieß mit deiner linken  hand wenn du linkshänder bist.",
+"GwCamera" : "Bewege dich in einen nicht sichtbaren punkt!",
+"GwCatch" : "Fang!",
+"GwClimb_Vr" : "Kletter zum dach!",
+"GwClimb_Desktop" : "Drücke und ziehe dich hoch um zu klettern!",
+"GwDevouringLurkerObstacleCourse" : "Flieh!",
+"GwDodgeLaser" : "Werde nicht von lasern getroffen!",
+"GwDontMoveAMuscle_VrStart" : "Beweg dich kein milimeter!",
+"GwDontMoveAMuscle_DesktopStart" : "Drücke linke und rechte Maustaste und bleib stehen!",
+"GwDontMoveAMuscle_VrContinue" : "Beweg dich!",
+"GwDontMoveAMuscle_DesktopContinue" : "Lass deine maus los!",
+"GwEatChips" : "Ess {0} chips!",
+"GwEVRC" : "Verlass VRChat!",
+"GwFindSignal" : "Finde das signal!",
+"GwHandsUp_DesktopDown" : "Guck runter!",
+"GwHandsUp_DesktopUp" : " Guck hoch!",
+"GwHandsUp_VrDown" : "Hände runter!",
+"GwHandsUp_VrUp" : "Hände hoch!",
+"GwHitBullseye_DontHit" : "Nicht schlagen!",
+"GwHitBullseye_1OrMore" : "schlag!",
+"GwHitBullseye_1Exactly" : "schlag NUR einmal!",
+"GwHitBullseye_NOrMore" : "schlag {0} mal!",
+"GwHitBullseye_NExactly" : "schlag exakt {0} mal!",
+"GwHitBullseye_Stop" : "Stop!",
+"GwHookFling" : "Ziehe dich zur schwarzen Plattform!",
+"GwInOrder_Start" : "Guck genau zu...",
+"GwInOrder_Continue" : "Schieß in der richtigen reihenfolge!",
+"GwJump" : "Spring!",
+"GwKeepCrabwalking" : "Lauf seitwärts und hör nicht auf!",
+"GwKeepMoving" : "Renn um dein leben!",
+"GwMostPlayers_Fewest" : "Blaues feld mit denn WENIGSTEN Spielern!",
+"GwMostPlayers_Most" : "Blaues feld mit denn MEISTEN Spielern!",
+"GwMostPlayers_CountingTie" : "Zählen... {0} VS {1} (Tie)",
+"GwMostPlayers_Counting" : "Zählen... {0} VS {1}",
+"GwRespawn" : "Respawn!",
+"GwRocketjump" : "Racketen sprung!",
+"GwShakeBottle" : "Shüttel die Flasche!",
+"GwSink" : "Versenk die Schwarze Kugel!",
+"GwSplitInOrder_Start" : "Erinerre dich...",
+"GwSplitInOrder_Incomplete" : "Warte bitte...",
+"GwSplitInOrder_Shoot" : "...Schieß in der selben Reihenfolge!",
+"GwStayAway_Start" : "Bleib weg von {0}!",
+"GwStayBetween_Start" : "Bleib zwischen {0} und {1}!",
+"GwAny_InterruptedByLeaving" : "{0} Hat verlassen",
+"GwAny_Minimum2Players" : "Manche spiele brauchen 2 spieler..\nLADE MEHR EIN!",
+"GwAny_Minimum5Players" : "Manche spiele brauchen 5 spieler..\nLADE MEHR FREUNDE EIN!",
+"GwTeamHitBullseye_Rifle" : "Leute, halte und schieße {0} mal!",
+"GwTeamHitBullseye_Pistol" : "Leute, halte und schieße {0} mal!",
+"GwTeamHitBullseye_Counting" : "Zeit ist vorbei! Zählen.. ({0} Treffer Fehlen.)",
+"GwTeamHitBullseye_Result" : "du hast {0} von {1} getroffen!",
```

Comment:
Please check this line and the others below to make sure that all sentences start with an uppercase letter